tests/sentry/web/frontend/test_auth_organization_login.py

```diff
@@ -1018,8 +1018,8 @@ def test_correct_redirect_as_2fa_user_invited(self) -> None:
             self.path, {"username": user, "password": "admin", "op": "login"}, follow=True
         )
 
-        invitation_link = "/" + member.get_invite_link().split("/", 3)[-1]
-        assert resp.redirect_chain == [(invitation_link, 302)]
+        # Users with 2FA should be redirected to 2FA dialog first, even with pending invites
+        assert resp.redirect_chain == [("/auth/2fa/", 302)]
```

Comment:
These were originally changed in https://github.com/getsentry/sentry/pull/82005/files#diff-8b70a9146a20fc2ca61609643e1c1aee1eac82c6c31a6ee75616997fbf5e715bL996-R1015